webapp/src/components/mobilitymap/LocationInstructionsAlert.js

```diff
@@ -0,0 +1,74 @@
+import api from "../../api";
+import { md, t } from "../../localization";
+import { useCurrentLanguage } from "../../localization/currentLanguage";
+import externalLinks from "../../modules/externalLinks";
+import ExternalLink from "../ExternalLink";
+import clsx from "clsx";
+import isEmpty from "lodash/isEmpty";
+import React from "react";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Stack from "react-bootstrap/Stack";
+
+const LocationInstructionsAlert = () => {
+  const [userAgent, setUserAgent] = React.useState({});
+  const [linkKey, setLinkKey] = React.useState("");
+  const [icon, setIcon] = React.useState("");
+  const [language] = useCurrentLanguage();
+  React.useEffect(() => {
+    if (!isEmpty(userAgent)) {
+      return;
+    }
+    api
+      .getUserAgent()
+      .then(api.pickData)
+      .then((ua) => {
+        // get the second word in case for "Microsoft Edge"
+        const browser = ua.device.toLowerCase().split(" ")[1] || ua.device.toLowerCase();
+        if (ua.isIos) {
```

Comment:
When do we see the browser-specific icons/instructions? This seems like we should be leading with the browser instructions, we can't know if the issue is the browser or the website.

Reply:
@rgalanakis I see what you mean. How about we always provide browser link if it's available, but also provide the device/platform (Android, IOS macOs, windows) link if we find one?

Reply:
I think we need to show both :( Unless the links cover both.

Reply:
@rgalanakis Good catch. The condition totally changed and made me realize that multiple externalLinks were created unnecessarily.
I added a commit for this.